src/benchmarklib/benchmark_runner.cpp

```diff
@@ -500,7 +500,8 @@ cxxopts::Options BenchmarkRunner::get_basic_cli_options(const std::string& bench
     ("visualize", "Create a visualization image of one LQP and PQP for each query, do not properly run the benchmark", cxxopts::value<bool>()->default_value("false")) // NOLINT
     ("verify", "Verify each query by comparing it with the SQLite result", cxxopts::value<bool>()->default_value("false")) // NOLINT
     ("dont_cache_binary_tables", "Do not cache tables as binary files for faster loading on subsequent runs", cxxopts::value<bool>()->default_value("false")) // NOLINT
-    ("metrics", "Track more metrics (steps in SQL pipeline, system utilization, etc.) and add them to the output JSON (see -o)", cxxopts::value<bool>()->default_value("false")); // NOLINT
+    ("metrics", "Track more metrics (steps in SQL pipeline, system utilization, etc.) and add them to the output JSON (see -o)", cxxopts::value<bool>()->default_value("false")) // NOLINT
+    ("data_preparation_cores", "Specify the number of cores used by the scheduler for the data preparation steps, i.e., sorting and encoding tables and generating table statistics. 0 means all available cores. This option is only advised when the underlying system's memory capacity is overleaded by the preparation phase.", cxxopts::value<uint32_t>()->default_value("0")); // NOLINT
```

Comment:
`overloaded`?

Reply:
Also, it is by far the longest description here.

Reply:
We could remove the information about 0, it's the same as with `--cores`. And the motivation "This option .." could also be a comment in the code. It will only be used by "experts" anyways.

What do you think?

Reply:
Yep, I would also remove the `This option...` part. Also, in the first part we could simply say `for data preparation` instead of `for the data preparation steps` to remove a few more characters.